docs/how-to-guides/eagleye-integration-guide.md

```diff
@@ -0,0 +1,140 @@
+# Eagleye Integration Guide
+
+This page will show you how to set up [eagleye](https://github.com/MapIV/eagleye), which is integrated into autoware.
+
+Discussions during the integration are as follows.
+https://github.com/orgs/autowarefoundation/discussions/3257
+
+## What is Eagleye?
+
+Eagleye provides a cost-effective alternative to LiDAR and point cloud-based localization by using low-cost GNSS and IMU sensors to provide vehicle position and attitude.
+Eagleye can also compensate for wheel speed and IMU angular velocity bias by using GNSS Doppler velocity.
+
+Autoware users will be able to choose between their existing LiDAR and point cloud-based localization stacks or GNSS/IMU-based eagleye localizer, depending on their specific needs and operating environment.
+
+There are two ways to utilize eagleye results with the autoware localization stack.
+
+1. Feed only Twist into the EKF localizer.
```

Comment:
For L7 - L9
```suggestion
1. Feed only twist into the EKF localizer.
![Eagleye twist integration](images/eagleye-integration-guide/eagleye-twist.png)
```
When there's a new line after a numbered list, the numbering doesn't seem to work properly. The numbers are shown as "1" and then "1" again. Please see the output documentation page below (this is your original):

![Screenshot from 2023-04-12 11-05-40](https://user-images.githubusercontent.com/23651560/231354245-c8857366-f391-4b2f-897e-bfce2fa872e4.png)

Also, I suggest changing Twist to twist to be more consistent.